### PR TITLE
ipq-wifi: update 8devices Habanero/Jalapeno BDFs

### PR DESCRIPTION
**Pull request by mantas-p**
Updatd BDFs fixes temperature sensor readout.

Boardfiles have been submitted upstream:
http://lists.infradead.org/pipermail/ath10k/2023-March/014297.html
http://lists.infradead.org/pipermail/ath10k/2023-March/014298.html

Also update MeshPoint.One as it uses Jalapeno SoM

### CODE REVIEW

File: target/linux/ipq40xx/image/generic.mk [hunk 1]
```diff
@@ -336,7 +338,7 @@ define Device/cilab_meshpoint-one
 	$(call Device/8dev_jalapeno-common)
 	DEVICE_VENDOR := Crisis Innovation Lab
 	DEVICE_MODEL := MeshPoint.One
-	DEVICE_PACKAGES := kmod-i2c-gpio kmod-iio-bmp280-i2c kmod-hwmon-ina2xx kmod-rtc-pcf2127
+	DEVICE_PACKAGES += kmod-i2c-gpio kmod-iio-bmp280-i2c kmod-hwmon-ina2xx kmod-rtc-pcf2127
```
Comment (BKPepe) [inline target/linux/ipq40xx/image/generic.mk:341]: I think this should be done in separate commit as it might seems that you are doing something else than what is said in the commit subject and commit description

Comment (mantas-p) [reply target/linux/ipq40xx/image/generic.mk:341]: You are right, I failed to explicitly mention MeshPoint.One thinking it as subclass of Jalapeno. Updated now.

**Review by BKPepe — APPROVED** [review]

**Comment by Ansuel** [comment]
@mantas-p BDF got moved in a separate repo can you resubmit there? https://github.com/openwrt/firmware_qca-wireless

(ignore that it's now empty... will be updated at the first chance when i update the git repo)

**Comment by mantas-p** [comment]
@Ansuel I don't think it's possible to make a PR to an empty repo. Or do you want me to send BDFs through the mailing list?

**Comment by BKPepe** [comment]
It should be possible, but still, this can go to the repository as it is, and it can be moved anytime there and it will be done as treewide commit, which is probably what we want. Let's not add more overhead to contributors.

**Comment by Ansuel** [comment]
Main problem is the makefile change so that is why I can't take this as is.

Il Gio 16 Mar 2023, 10:09 Josef Schlehofer ***@***.***> ha
scritto:

**Comment by mantas-p** [comment]
@Ansuel I've rebased this, but I can't make a PR to https://github.com/openwrt/firmware_qca-wireless because it's empty. Can you just take changes from: https://github.com/mantas-p/qca-wireless ?

**Comment by Ansuel** [comment]
@mantas-p done!

**Comment by Ansuel** [comment]
Now you should bump the ipq-wifi package version and you should be good to go.

**Comment by Ansuel** [comment]
@mantas-p i found time to fix this and I split 2 commits. 

If the changes I did are ok for you I will merge this. (asking since i split the commit and still used you as author so asking permission for the change)

**Comment by mantas-p** [comment]
@Ansuel thanks, looks good, please merge.
